librz/diff/distance.c

```diff
@@ -66,7 +66,9 @@ RZ_API bool rz_diff_myers_distance(RZ_NONNULL const ut8 *a, ut32 la, RZ_NONNULL
  * - similarity: is a number that defines how similar/identical the 2 buffers are.
  * */
 RZ_API bool rz_diff_levenstein_distance(RZ_NONNULL const ut8 *a, ut32 la, RZ_NONNULL const ut8 *b, ut32 lb, RZ_NULLABLE ut32 *distance, RZ_NULLABLE double *similarity) {
-	rz_return_val_if_fail(a && b, false);
+	if (!a || !b) {
+		return false;
+	}
```

Comment:
why? if it's NONNULL then rz_return shall be used and the caller should be fixed.

Reply:
`rz_return_val_if_fail` will print a warn message. It has no problem in most cases.
But https://github.com/rizinorg/rizin/runs/7917197046?check_suite_focus=true

It looks like stderr is mixed into stdout, in some tests

![image](https://user-images.githubusercontent.com/24983369/185756558-215b1f31-3edb-43b0-9450-d0b7fe1840fd.png)

Reply:
I suggest also creating an issue for this to not block this PR. What do you think @ret2libc?

Reply:
Just check the parameters on the caller side. There are some new places (or some changes in this PR which trigger this) that calls this function with NULLs when they should not be.
I think it's better to "fix" it here, considering it was not happening before.

Reply:
this is wrong, that function must have non zero values.